game-core/src/main/java/games/strategy/engine/random/PbemDiceRoller.java

```diff
@@ -133,7 +134,7 @@ private static Frame getFocusedFrame() {
       this.diceServer = diceServer;
       this.gameUuid = gameUuid;
       setDefaultCloseOperation(WindowConstants.DO_NOTHING_ON_CLOSE);
-      exitButton.addActionListener(e -> System.exit(-1));
+      exitButton.addActionListener(e -> ExitStatus.FAILURE.exit());
```

Comment:
BTW, forgot to call this out just as a general FYI...

I found it strange that we offer the user the ability to terminate the application from the PBEM/PBF dice roller dialog (this button is only enabled if the dice server signals an error).  Not sure what the history is on that.